http2_general/test_h2_hpack.py

```diff
@@ -152,6 +152,36 @@ def test_settings_header_table_size(self):
             "Tempesta encode large header, but HEADER_TABLE_SIZE smaller than this header.",
         )
 
+    def test_relloc_hpack_table(self):
+        """
+        When count of entries in hpack dynamic table exceeded it's size
+        Tempesta FW realloc hpack dynamic table. This test check it.
+        """
+        self.start_all_services()
+        client: DeproxyClientH2 = self.get_client("deproxy")
+        client.parsing = False
+
+        headers = [
+            HeaderTuple(":path", "/"),
+            HeaderTuple(":scheme", "https"),
+            HeaderTuple(":method", "POST"),
+        ]
+
+        for i in range(0, 125):
+            for j in range(0, 26):
+                key = ord('a') + j
+                val = ord('a') + j
+
+                first_indexed_header = [HeaderTuple(chr(key) * (125 -i), chr(val) * (125 - i))]
+                client.send_request(
```

Comment:
most likely, we'd better use `make_request` here. This will speed up the test execution